### PR TITLE
[schema] Update dockerhub.csv

### PR DESCRIPTION
**Pull request by vchrombie**
This PR updates the dockerhub schema to the latest format.
Reference to #803

### CODE REVIEW

File: schema/dockerhub.csv [hunk 1]
```diff
@@ -17,10 +17,9 @@ metadata__timestamp,date
 metadata__updated_on,date
 origin,keyword
 pull_count,long
-repository_labels,keyword
```
Comment (valeriocos) [inline schema/dockerhub.csv]: please keep it. This appears when labeling a repo (more info at: https://github.com/chaoss/grimoirelab-sirmordred#projectsjson-, look for `--labels`)

**Review by valeriocos — COMMENTED** [review]
It looks good @vchrombie, thank you for your effort. Would be possible to add the description of each field (something similar to https://github.com/chaoss/grimoirelab-elk/blob/master/schema/git.csv)?

Please note that some descriptions can be taken from existing CSVs

File: schema/dockerhub.csv (hunk 1 above)
Comment (vchrombie) [reply schema/dockerhub.csv]: Sure, I will check for these things in the coming PRs.
Thanks

Comment (vchrombie) [reply schema/dockerhub.csv]: Updated.

**Comment by vchrombie** [comment]
Hi @valeriocos 
The `dockerhub` schema is completed. It is ready for review.
It has 3 commits as of now.
1. https://github.com/chaoss/grimoirelab-elk/pull/814/commits/3848aeeab9479ecf2b4d870d87f87d73399c24f5, which updates all the fields with the latest index
2. https://github.com/chaoss/grimoirelab-elk/pull/814/commits/4446298e8626238e4542158cc38767b1731be34e, which updates the `aggregatable` column of the fields
3. https://github.com/chaoss/grimoirelab-elk/pull/814/commits/41bf585570a09282c96fc91764ea9bc7747aa57a, which updates the `description` column of the field (this might need a review :sweat_smile: )

Please review it when you are free. After the PR is approved, I will squash all these commits into 1.
Thanks. :slightly_smiling_face:

**Review by valeriocos — APPROVED** [review]
LGTM, thanks @vchrombie! good job!

Please squash the commits and ping me when I can merge the PR.

**Comment by vchrombie** [comment]
Hi @valeriocos 

I have squashed the commits too. Please let me know if there are any more changes to do. :slightly_smiling_face:

**Comment by valeriocos** [comment]
Thank you @vchrombie !

**Pull request merged**